### PR TITLE
pep8 changes and Travis testing

### PR DESCRIPTION
**Pull request by bgruening**
This PR will make the moFF tools [pep8](https://www.python.org/dev/peps/pep-0008/) compliment and enables initial Travis testing - pep8 tests and a calling the main script.
Would be great if we could enable functional testing as well with some very small example datasets.

### CODE REVIEW

**Comment by bgruening** [comment]
Sorry that the diff is so large and not so good to investigate. It turned out that the `moff.py` script had some nasty CR line endings which needed to be removed.

**Comment by bgruening** [comment]
I added the ability to overwrite the txic path via an environment variable (`TXIC_PATH`) and fixed a bug in finding the RAW files on unix.

**Comment by bgruening** [comment]
A package with all these changes can be installed via conda with:

```
conda install python-moff --channel bgruening
```

I'm planning to create a bioconda package if you think these changes are ok to merge.
Thanks!

**Comment by Maux82** [comment]
I have some questions : 
The env. variable TXIC_path that contains the txic path , how is initialized if the user just run the script in the command line mode outside the bioconda package ? 

You  replaced all my prints to the console with the print to log file. Now all the prints are  in the log file ? or some also to the console ?

**Comment by Maux82** [comment]
I have some questions : 
The env. variable TXIC_path that contains the txic path , how is initialized if the user just run the script in the command line mode outside the bioconda package ? 

You  replaced all my prints to the console with the print to log file. Now all the prints are  in the log file ? or some also to the console ?

For the rest is seems fine by me. You cleaned a lot my code  thanks :+1:

File: moff.py [hunk 1]
```diff
@@ -1,313 +1,300 @@
...
+    log.info('moff Input file: %s  XIC_tol %s XIC_win %4.4f moff_rtWin_peak %4.4f ', file_name, tol, h_rt_w, s_w)
+    log.info('Output_file in :  %s', outputname)
+    log.info('RAW file and its location :  %s', loc)
+    # read data from file
+    data_ms2 = pd.read_csv(file_name, sep="\t", header=0)
+    if check_columns_name(data_ms2.columns.tolist(), ast.literal_eval(config.get('moFF', 'col_must_have_x'))) == 1:
+        exit('ERROR minimal field requested are missing or wrong')
+
+    # fast debug ---
+    data_ms2 = data_ms2.head(100)
```
Comment (Maux82) [inline moff.py]: this line should be deleted . I used to load only the first 100 features for a fast debug

**Comment by bgruening** [comment]
> I have some questions : 
> The env. variable TXIC_path that contains the txic path , how is initialized if the user just run the script in the command line mode outside the bioconda package?

It defaults to `./` in this line: https://github.com/compomics/moFF/pull/1/files#diff-9ef8fa1a4fe6d91b8bbc5688659ab5baR25
So actually it should not change the behavior of your script but it includes an entry point where I can override this path. Ideally this should default to ``(txic is on your PATH) and not`./` (your current working directory). But I wanted to discuss this first with you :) 

> You replaced all my prints to the console with the print to log file. Now all the prints are in the log file ? or some also to the console ?

This depends on how you set up your logging in python. I have seen that one script actually printed to a file but other not. I tried to streamline this. All output should go now through the logging module and I hope there is no print anymore.
I admit this area needs additional love or at least discussion how this should be solved. I'm not against printing to stdout as long as it goes though `log.*` :)

> For the rest is seems fine by me. You cleaned a lot my code thanks :+1:

Thanks for the fast review, I hope I did not break anything.

**Comment by bgruening** [comment]
Galaxy wrapper is also working. If you think this is ok for merging I will go ahead and create a bioconda package and submit the Galaxy wrapper to GalaxyP.
Thanks!

**Pull request merged**